src/cascadia/TerminalControl/TermControl.cpp

```diff
@@ -1792,6 +1792,11 @@ namespace winrt::Microsoft::Terminal::TerminalControl::implementation
         return std::pow(cursorDistanceFromBorder, 2.0) / 25.0 + 2.0;
     }
 
+    bool TermControl::HasSelection() const
+    {
+        return _terminal != nullptr && _terminal->IsAreaSelected();

```

Comment:
there's also `isSelectionActive`. @carlos-zamora can you clarify which to use here?

Reply:
OMG. THEY'RE BOTH THE SAME. Adding to #1327.